include/quda.h

```diff
@@ -1214,37 +1214,26 @@ extern "C" {
    * @param _hp_x       Array of solution spinor fields
    * @param _hp_b       Array of source spinor fields
    * @param param       Contains all metadata regarding host and device storage and solver parameters
-   * @param h_gauge     Base pointer to host gauge field (regardless of dimensionality)
-   * @param gauge_param Contains all metadata regarding host and device storage for gauge field
    */
-  void invertMultiSrcQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param, void *h_gauge, QudaGaugeParam *gauge_param);
+  void invertMultiSrcQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param);
 
   /**
    * @brief Really the same with @invertMultiSrcQuda but for staggered-style fermions, by accepting pointers
    * to fat links and long links.
    * @param _hp_x       Array of solution spinor fields
    * @param _hp_b       Array of source spinor fields
    * @param param       Contains all metadata regarding host and device storage and solver parameters
-   * @param milc_fatlinks     Base pointer to host **fat** gauge field (regardless of dimensionality)
-   * @param milc_longlinks    Base pointer to host **long** gauge field (regardless of dimensionality)
-   * @param gauge_param Contains all metadata regarding host and device storage for gauge field
    */
-  void invertMultiSrcStaggeredQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param, void *milc_fatlinks,
-                                   void *milc_longlinks, QudaGaugeParam *gauge_param);
+  void invertMultiSrcStaggeredQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param);
```

Comment:
Do we even need separate interface nows since we can use the resident gauge / fat / long / clover fields?  Can't we delete all these?

Reply:
Possibly...